test/wdt_e2e_simple_test.sh

```diff
@@ -49,6 +49,15 @@ if [ -z "$WDT_TEST_SYMLINKS" ] ; then
 fi
 echo "WDT_TEST_SYMLINKS=$WDT_TEST_SYMLINKS"
 
+# Set WDT_TEST_PERMISSION:
+# to 1 : check if permissions match
+# to 0 : when "find -printf %m" does not work, i.e. on Mac
+if [ -z "$WDT_TEST_PERMISSION" ] ; then
+  WDT_TEST_PERMISSION=1
+  umask 0
```

Comment:
I guess you are asking about why umask? Some machines (including mine) have a mask by default, which prevents giving write permission to Others. If we do not umask it, the file created on the receiver side will not have the write perm even if it is a 0777 on the writer side.